README.md

```diff
@@ -121,6 +121,42 @@ handleCastEvent(args): void {
 }
 ```
 
+#### Angular
+
+Add the `NativeScriptModule` in your app module's `imports` 
+
+Typically in `app.module.ts`
+```ts
+import {NativescriptCastModule} from 'nativescript-cast/angular';
+
+@NgModule({
+    imports: [NativeScriptModule]
```

Comment:
Should this be `NativescriptCastModule` ?

Reply:
yes.. copy paste mistake. Perhaps I should just add a complete `angular-demo` folder with a working app, then that could be done later for Vue as well.